docs/KIALI.md

```diff
@@ -0,0 +1,37 @@
+## Kiali integration
+
+This server can expose Kiali tools so assistants can query mesh information (e.g., mesh status/graph).
+
+### Enable the Kiali toolset
+
+Enable the Kiali tools via the server TOML configuration file.
+
+Config (TOML):
+
+```toml
+toolsets = ["core", "kiali"]
+
+[toolset_configs.kiali]
+url = "https://kiali.example"
```

Comment:
@aljesusg maybe it is because I haven't used Kiali much but it's not clear to me from the docs here what value should go in this url/how I would find it.

Would you mind adding a sentence somewhere about what this value should be?

Reply:
thanks!